### PR TITLE
[WIP] Windows support

### PR DESCRIPTION
**Pull request by axos88**
### Description

Windows Support

Includes PR #41, which was closed prematurely.

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by axos88** [comment]
Not sure why commits 8670e24 and 12cc5cc are included, as they have already been merged.

File: metadata.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@
 depends 'compat_resource', '>= 12.14.6'
 depends 'zypper'
 
+suggests 'nssm'
```
Comment (tas50) [inline metadata.rb]: suggests was never fully implemented and we're working to fully deprecate the functionality in chef. This shouldn't be use here

Comment (axos88) [reply metadata.rb]: Hm. How would you define a dependency that is needed for one operating system, but not another? Using depends seems like would include unneeded dependencies on some platforms.

Comment (tas50) [reply metadata.rb]: You just depends on it and pull in the deps

Comment (axos88) [reply metadata.rb]: done

**Comment by axos88** [comment]
rebased on top of master.

**Comment by axos88** [comment]
@tas50 Any other comments? I will rebase on top of current master, once it is in an LGTM state.

**Comment by axos88** [comment]
@tas50

**Comment by iennae** [comment]
Hi,

This cookbook will be deprecated in favor of the nginx cookbook. https://github.com/chef-cookbooks/nginx. Please open up this PR on that cookbook. 

Thanks.

**Pull request closed without merging**